Source/JavaScriptCore/wasm/WasmLLIntTierUpCounter.h

```diff
@@ -61,7 +61,8 @@ class LLIntTierUpCounter : public BaselineExecutionCounter {
 
     void optimizeAfterWarmUp()
     {
-        setNewThreshold(Options::thresholdForBBQOptimizeAfterWarmUp());
+        setNewThreshold(Options::useWasmLLInt() ? Options::thresholdForBBQOptimizeAfterWarmUp() : 0);
```

Comment:
Let's change these numbers in Options.cpp when useWasmLLInt is set instead of changing it here.